content/docs/guides/node-exporter.md

```diff
@@ -0,0 +1,208 @@
+
+```bash
+go get github.com/prometheus/node_exporter
+```
+
+This installs the `node_exporter` executable in the `$GOPATH/bin` folder. To run the node exporter on port 9100 (the default):
+
+```bash
+node_exporter
+```
+
+That should produce output like this:
+
+```
+INFO[0000] Starting node_exporter (version=, branch=, revision=)  source="node_exporter.go:82"
+INFO[0000] Build context (go=go1.10.3, user=, date=)     source="node_exporter.go:83"
+INFO[0000] Enabled collectors:                           source="node_exporter.go:90"
+INFO[0000]  - boottime                                   source="node_exporter.go:97"
+...
+INFO[0000] Listening on :9100                            source="node_exporter.go:111"
+```
+
+### Docker
+
+There is an official [Docker](https://docker.com) image for the node exporter available via [Docker Hub](https://hub.docker.com/r/prom/node-exporter/) as `prom/node-exporter`. You can see a list of available tags [here](https://hub.docker.com/r/prom/node-exporter/tags/). To run the latest version of the image locally:
+
+```bash
+docker run \
+  -p 9100:9100 \
+  prom/node-exporter:latest
+```
+
+## Node exporter metrics
+
+Once the node exporter is installed and running, you can verify that metrics are being exported by cURLing the `/metrics` endpoint:
+
+```bash
+curl http://localhost:9100/metrics
+```
+
+You should see [PromQL](/docs/prometheus/latest/querying/basics)-formatted output like this:
+
+```
+# HELP go_gc_duration_seconds A summary of the GC invocation durations.
+# TYPE go_gc_duration_seconds summary
+go_gc_duration_seconds{quantile="0"} 3.8996e-05
+go_gc_duration_seconds{quantile="0.25"} 4.5926e-05
+go_gc_duration_seconds{quantile="0.5"} 5.846e-05
+# etc.
+```
+
+Success! The node exporter is now exposing a wide variety of system metrics that Prometheus can scrape.
+
+## Node exporter run options
+
+By default the Prometheus node exporter is available via host `:9100` and endpoint `/metrics`. You can change that using the `--web.listen-address` flag:
+
+```bash
+node_exporter --web.listen-address=9101
+```
+
+The path of the metrics-exposing endpoint (`/metrics` by default) can be changed using the `--web.telemetry-path` flag:
+
+```bash
+node_exporter --web.telemetry-path="/prometheus-metrics"
+```
+
+## Configuring your Prometheus instances
+
+Your locally running Prometheus instance needs to be properly configured in order to access node exporter metrics. The following [`scrape_config`](../prometheus/latest/configuration/configuration/#<scrape_config>) block will tell Prometheus that scrape from the node exporter via `localhost:9100`:
+
+```yaml
+scrape_configs:
+- job_name: 'node_exporter'
+  scrape_interval: 5s
+  static_configs:
+  - targets: ['localhost:9100']
+```
+
+Once Prometheus is [installed](../../introduction/first_steps) you can start it up, using the `--config.file` flag to point to the Prometheus configuration that you created:
```

Comment:
Probably best to be self-contained

Reply:
It'd be good to explain how to obtain prometheus. We can skip it for most of the others, but the Node exporter is likely the first thing a user will use